### DISCUSSION TITLE
It seems like `Hook (Future e a) b` is equivalent to `ContT a (Future e) b`

### DISCUSSION THREAD

**Discussion by safareli** [fluture-js/Fluture#465] [General]
I was looking at [this lib](https://github.com/fluture-js/fluture-hooks) and noticed that the Hook structure is Continuation monad.

**Comment by Avaq** [comment]
[Fluture Hooks](https://github.com/fluture-js/fluture-hooks) is essentially a tagged constructor (with FL methods) wrapper around [callgebra](https://github.com/fluture-js/callgebra), which I believe is just an implementation of the continuation Monad.

The only thing that makes it Fluture -specific is the Apply instance on `ParallelHook (Future e a) b`:

https://github.com/fluture-js/fluture-hooks/blob/e2fd32742a3b1bc9cd0922ff1610395af8800409/index.js#L158-L226

**Reply by safareli** [reply]
Hi, I closed the issue as it was not actionable.

**Reply by Avaq** [reply]
Ah, no worries. That's why I transferred it to a Discussion. I think it's worth sharing this info.
